### PR TITLE
Omega TFT

### PR DESCRIPTION
**Pull request by marcharper**
Adds the OmegaTFT strategy that won a 2004 tournament, described [here](http://arxiv.org/pdf/cs/0609017.pdf).

### CODE REVIEW

**Comment by langner** [comment]
+1

File: docs/tutorials/contributing/strategy/writing_the_new_strategy.rst [hunk 1]
```diff
@@ -7,7 +7,7 @@ Identify a new strategy
 The library has a large number of strategies::
 
     >>> import axelrod as axl
-    >>> len(axl.strategies)
+    >>> len(axl.strategies) # doctest: +SKIP
```
Comment (drvinceknight) [inline docs/tutorials/contributing/strategy/writing_the_new_strategy.rst]: Same comment here as on #388

File: axelrod/strategies/titfortat.py [hunk 2]
```diff
@@ -180,3 +180,65 @@ def strategy(opponent):
             return D
         # Otherwise cooperates
         return C
+
+class OmegaTFT(Player):
```
Comment (drvinceknight) [inline axelrod/strategies/titfortat.py]: Can we have a docstring here.

**Comment by marcharper** [comment]
Will rebase once #388 is in...

**Pull request merged**